shaka/src/memory/object_tracker.cc

```diff
@@ -54,14 +54,16 @@ void ObjectTracker::RemoveRef(const Traceable* object) {
     std::unique_lock<Mutex> lock(mutex_);
     auto* key = const_cast<Traceable*>(object);  // NOLINT
     DCHECK_EQ(objects_.count(key), 1u);
-    CHECK_GT(objects_[key], 0u);
-    objects_[key]--;
+    uint32_t& ref_count = objects_.at(key);
```

Comment:
Non-const references are discouraged by the style guide.  You could use `find` instead.  That would also allow you to change the `DCHECK_EQ` to avoid another lookup there.